pkgs/servers/x11/xarcan/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, lib, fetchFromGitHub, meson, pkg-config, xorgproto, xtrans
+, libxshmfence, pixman, xkbcomp, libxkbfile, libXfont2, dbus, systemd, fontutil
+, mesa, epoxy, libmd, nettle, libgcrypt, openssl, libXdmcp, libdrm, libselinux
+, audit, libGL, libtirpc, libXau, arcan, libxcb, ninja, libX11 }:
+
+stdenv.mkDerivation rec {
+  pname = "xarcan";
+
+  version = "0.6.0+unstable=2021-05-27";
+
+  src = fetchFromGitHub {
+    owner = "letoram";
+    repo = pname;
+    rev = "3dc5f8114bef88104e93276c2ccd957c0789df3a";
+    sha256 = "sha256-zlk3Jw3YVW59AvWwD8qZJTujPgsKSxUK068bgpkhHl0=";
+  };
+
+  nativeBuildInputs = [ meson pkg-config ];
+
+  buildInputs = [
+    xorgproto
+    xtrans
+    libxshmfence
+    pixman
+    xkbcomp
+    libxkbfile
+    libXfont2
+    dbus
+    systemd
+    fontutil
+    mesa
+    epoxy
+    libmd
+    nettle
+    libgcrypt
+    openssl
+    libXdmcp
+    libdrm
+    libselinux
+    audit
+    libGL
+    libtirpc
+    libXau
+    arcan
+    libxcb
+    ninja
+    libX11
+  ];
+
+  meta = with lib; {
+    description = "Patched Xserver that bridges connections to Arcan";
+    longDescription = "";
+    homepage = "https://github.com/letoram/${pname}";
+    changelog = "https://github.com/letoram/${pname}/releases/tag/${version}";
```

Comment:
If you are using the master branch and not a tagged one, then the changelog should point out to the master branch, not to a particular version.

However, master branches have no consolidated changelog. Just erase this line.

```suggestion
```

Reply:
Using the masterbranch is a temporary.